libs/ml/tests/unit/layers/scaled_dot_product_attention.cpp

```diff
@@ -198,13 +198,13 @@ TYPED_TEST(ScaledDotProductAttention,
   g.SetInput(mask, mask_data);
 
   TypeParam gt = TypeParam::FromString(
-      "1.8496745531,  1.9944926680,  1.5288354812,  0.1000000000, 0.1000000000,  0.1000000000; "
-      "1.1503254469,  1.0055073320,  1.4711645188, -0.2000000000, -0.2000000000, -0.2000000000; "
-      "3.6993491062,  3.9889853359,  3.0576709623,  1.0000000000, 1.0000000000,  1.0000000000");
+      "+1.848551915, +1.994484604, +1.323288694, +2.134702830, +1.607807817, "
+      "+3.647750064;+1.149618159, +1.005503525, +1.237182403, -0.486945271, -0.412639564, "
+      "-0.700323727;+3.696272045, +3.988963812, +2.446791525, -0.017351415, +0.246096091, "
+      "-0.773875033;");
```

Comment:
This seems weird because mask should block gradient, so these values  0.1000000000, 0.1000000000,  0.1000000000; etc. should be the same.